### PR TITLE
Upgrade richie to v2.0.0 beta.7

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

We want to upgrade all sites to the new richie pre-release version [2.0.0-beta.7](https://github.com/openfun/richie/releases/tag/v2.0.0-beta.7)

## Proposal

- [x] Bump all sites requirements
- [x] Make a new release of each site's image
- [x] Improve circleCI configuration to always check that tags, versions and CHANGELOG are in sync.

### CODE REVIEW

File: sites/funcorporate/src/frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "funcorporate",
-  "version": "0.2.2",
+  "version": "0.3.0",
```
Comment (jmaupetit) [inline sites/funcorporate/src/frontend/package.json:3]: Ditto.

File: sites/demo/src/frontend/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "richie",
-  "version": "2.0.0-beta.6",
+  "version": "2.0.0-beta.7",
```
Comment (jmaupetit) [inline sites/demo/src/frontend/package.json:3]: Ditto.

File: sites/cnfpt/src/frontend/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cnfpt",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (jmaupetit) [inline sites/cnfpt/src/frontend/package.json:3]: Ditto.

File: bin/ci [hunk 4]
```diff
@@ -103,28 +103,36 @@ function check_tag() {
...
+
     init_path="${site_path}/src/backend/${site}/__init__.py"
     if ! grep "__version__ = \"${version}\"" "${init_path}" &> /dev/null ; then
       echo "Backend version does not match the current tag!"
       fail=4
     fi
+
     if ! grep "\"version\": \"${version}\"" "${site_path}/src/frontend/package.json" &> /dev/null ; then
       echo "Frontend version does not match the current tag!"
       fail=4
```
Comment (jmaupetit) [inline bin/ci:133]: Different failures should correspond to different exit codes.

File: bin/ci [hunk 5]
```diff
@@ -103,28 +103,36 @@ function check_tag() {
...
-    if ! [[ "${tag}" =~ ${site}- ]] ; then
-      echo "Tags must be of the form \"{site}-{semver version}\"!"
-      fail=4
+    if [[ -n "${tag}" ]] ; then
+        # Tags should be of the form "demo-1.2.1" in which "1.2.1" is the version
+        if ! [[ "${tag}" =~ ${site}- ]] ; then
+            echo "Tags must be of the form \"{site}-{semver version}\"!"
+            fail=4
+        fi
+        version=${2//${1}-/}
```
Comment (jmaupetit) [inline bin/ci:114]:
```suggestion
        version=${tag//${site}-/}
```

File: bin/ci (hunk 4 above)
Comment (sampaccoud) [reply bin/ci]: I found a way to always be sure the returned code is unique for each different case :shrimp:

File: bin/ci (hunk 5 above)
Comment (sampaccoud) [reply bin/ci]: :+1:

**Pull request merged**